core/src/test/scala/matryoshka/spec.scala

```diff
@@ -402,6 +402,34 @@ class FixplateSpecs extends Specification with ScalaCheck with ScalazMatchers {
       }
     }
 
+    def depth[T[_[_]], F[_]]: (Int, T[F]) => Int = (i, t) => i + 1
+
+    def sequential[T[_[_]], F[_]]: (Int, T[F]) => State[Int, Int] =
+      (x, y) => State[Int, Int](prev => (prev + 1, prev))
```

Comment:
Similarly, `(_, _)` would work here. Also, it might be cleaner to do something like `State.get <* State.modify(_+1)`, but maybe not when you add the types to make it compile.